boards/opentitan/src/main.rs

```diff
@@ -220,6 +221,11 @@ impl KernelResources<earlgrey::chip::EarlGrey<'static, EarlGreyDefaultPeripheral
     }
 }
 
+#[inline(always)]
+fn div_ceil(a: u32, b: u32) -> u32 {
```

Comment:
It's worth adding a comment that we can remove this and use the Rust standard one at some point.

See: https://doc.rust-lang.org/std/primitive.i32.html#method.div_ceil and https://github.com/rust-lang/rust/issues/88581